content/lessons/genesis/genesis.tsx

```diff
@@ -13,7 +13,6 @@ import {
   LessonPrompt,
 } from 'ui'
 
-const inputAmount = 154
 const answer =
   '04ffff001d0104455468652054696d65732030332f4a616e2f32303039204368616e63656c6c6f72206f6e206272696e6b206f66207365636f6e64206261696c6f757420666f722062616e6b73'
```

Comment:
I am just grabbing it from the answer posted in the genesis content page, I am going to try and make catch to make sure that only numbers/characters can be pasted in correctly